docs/changelog.md

```diff
@@ -1,13 +1,105 @@
 <h1>Change Log</h1>
 
+## 1.0.0
+
+* More consistent template variables without globals (#394)
+* Auto fit search bar around text (#407)
+* Add link to Open SDG to footer (#411)
+* Configurable footer menu links (#414)
+* Escape variables in alt attributes (#415)
+* Use indicator_available instead of graph_title for the indicator page H2 (#422)
+* Reporting status for any metadata field (#424)
+* Switch to Lunr for search functionality (#427)
+* Metadata field translation keys (#428)
+* Show post excerpts on news page (#431)
+* Sass refactor (#439)
```

Comment:
"Refactoring of Sass style sheets" might be more descriptive

Reply:
Makes sense.